src/store/actions/tutorialsActions.js

```diff
@@ -130,44 +129,36 @@ export const createTutorial =
 
       const setData = async (type) => {
         const document = firestore
-          .collection("cl_codelabz")
-          .doc(type)
-          .collection(owner)
-          .doc();
+          .collection("tutorials")
+          .doc()
 
         const documentID = document.id;
         const step_id = `${documentID}_${new Date().getTime()}`;
 
         await document.set({
-          created_by,
-          owner,
-          summary,
-          title,
-          featured_image: "",
-          icon: "",
-          url: "",
-          steps: {
-            [step_id]: {
-              id: step_id,
-              title: "Step One Title",
-              time: 1,
-              content: "Sample tutorial step one",
-              visibility: true,
-              deleted: false,
-            },
-          },
-          background_color: "#ffffff",
-          text_color: "#000000",
-          createdAt: firestore.FieldValue.serverTimestamp(),
-          updatedAt: firestore.FieldValue.serverTimestamp(),
-        });
+            created_by,
+            editors: [created_by],
+            owner,
+            summary,
+            title,
+            featured_image: "",
+            icon: "",
+            url: "",
```

Comment:
Greatt work so far!!
Just a doublt, 
Is there any way to ensure that this post is draft/archived/published ? If not, then we can add these attributes to the set.

Else how are are handling draft & published posts right now

Reply:
I somehow missed it. Thanks for pointing out!
I implemented it in the following commit